conformance/lang/expressions/new-expr/explicit-new-expr/stream_class_descriptor.balt

```diff
@@ -0,0 +1,167 @@
+Test-Case: error
+Description: Test new-expression to create a stream without type descriptors.
+Labels: new-expr, stream-type-descriptor
+
+function init() {
+    var _ = new stream<>(); // @error missing type descriptor
+}
+
+Test-Case: error
+Description: Test new-expression to create a stream with incompatible type-descriptor.
+Labels: new-expr, stream-type-descriptor
+
+function init() {
+    stream<int> _ = new stream<string>(); // @error incompatible types: expected 'stream<int>', found 'stream<string>'
+}
+
+Test-Case: output
+Description: Test new-expression to create a stream with incompatible type-descriptor.
+Fail-Issue: ballerina-platform/ballerina-lang/issues/34963
+Labels: new-expr, stream-type-descriptor
+
+function init() {
+    stream s = new stream<string, error?>();
+    io:println(s); // @output stream <stream<int,error?>>
+}
+
+Test-Case: output
+Description: Test new-expression to create a empty streams.
+Labels: new-expr, stream-type-descriptor, int-type-descriptor
```

Comment:
int is in the minimal subset. missing labels: record-type, is-expr, if-else-stmt, module-type-defn, error-type, optional-type, union-type. check other places also.

Reply:
Thanks I think I have not focused on labels